pkg/requests/base.go

```diff
@@ -93,6 +95,7 @@ func (rb *Base) InitFlags(includeData bool) {
 	rb.Cmd.Flags().BoolVarP(&rb.showHeaders, "show-headers", "s", false, "Show headers on responses to GET, POST, and DELETE requests")
 	rb.Cmd.Flags().BoolVarP(&rb.autoConfirm, "confirm", "c", false, "Automatically confirm the command being entered. WARNING: This will result in NOT being prompted for confirmation for certain commands")
 	rb.Cmd.Flags().BoolVar(&rb.Livemode, "livemode", false, "Make a request against live mode instead of test mode")
+	rb.Cmd.Flags().BoolVar(&rb.DarkStyle, "dark-style", false, "Use a darker color scheme better suited for lighter terminals")
```

Comment:
Don't feel too strongly on this, but what do you think about `--theme dark` or something in case you decide to add more themes in the future?

Reply:
It feels a little weird to me to have a flag with just one text option right now so I'm thinking of keeping the bool. It should be easy for us to map this to a theme in the future if we need to!